package/src/components/AttachmentPicker/components/AttachmentSelectionBar.tsx

```diff
@@ -65,6 +65,7 @@ export const AttachmentSelectionBar: React.FC = () => {
   return (
     <View style={[styles.container, container, { height: attachmentSelectionBarHeight ?? 52 }]}>
       <TouchableOpacity
+        accessibilityLabel='upload-photo-touchable'
```

Comment:
This sounds wrong. 

https://reactnative.dev/docs/accessibility#accessibilitylabel

> When a view is marked as accessible, it is a good practice to set an accessibilityLabel on the view, so that people who use VoiceOver know what element they have selected. VoiceOver will read this string when a user selects the associated element.

Reply:
Ideally it should be something sensible that voice over can read!!

Reply:
But adding these labels only for 2-3 views  for the sake of testing doesn't sound good as well.

Reply:
And seems like RNGH 2.3.1 is out ;)

Reply:
https://github.com/software-mansion/react-native-gesture-handler/releases

Reply:
Done!